packages/video_player/video_player_platform_interface/lib/messages.dart

```diff
@@ -43,7 +45,8 @@ class CreateMessage {
       ..asset = pigeonMap['asset'] as String?
       ..uri = pigeonMap['uri'] as String?
       ..packageName = pigeonMap['packageName'] as String?
-      ..formatHint = pigeonMap['formatHint'] as String?;
+      ..formatHint = pigeonMap['formatHint'] as String?
+      ..httpHeaders = pigeonMap['httpHeaders'] as Map<Object?, Object?>?;
```

Comment:
I haven't used pigeon myself yet; does this need to be Object rather than String?

Reply:
As far as I understand, pigeon doesn't support generics at the moment. httpHeaders field is actually [declared](https://github.com/flutter/plugins/blob/0761edf6402e8b3e1321effb003cf21b92262490/packages/video_player/video_player/pigeons/messages.dart#L34) as `Map<String, String>` in pigeons/messages.dart. But lib/messages.dart is generated so if we follow "do not manually edit generated files" rule, we should keep it as is